### ISSUE TITLE
State Parks website - map redesign

### ISSUE THREAD

**Issue by chriswnek** [agrc/planning-queue#939]
### Benefit

State Parks is working with the DPX team on a redesign of their website.  As part of this project, UGRC will work on designing a new and accessible basemap, as well as building more usability and modernization into their interactive maps.

A second phase of this project will also include partnering with Weber State University to create printable maps for all state parks.

### Acceptance Criteria

- [ ] Parksy basemap development
- [ ] Zoom extent fixes for all 49 parks
- [ ] Labels must have a minimum 4:5:1 Contrast ratio
  - Adding a halo around labels will help with this
  - Also see: [esri Accessibility best practices](https://pro.arcgis.com/en/pro-app/latest/help/analysis/business-analyst/accessibility-best-practices.htm)
- [ ] New skid/WordPress web hooks

### Notes

- <ins>Vantage project task:</ins> **DNR-SP Website** Interactive Maps || PRJTASK0701413
- [GIS accessibility resources](https://docs.google.com/document/d/1igWU7F4TevLGJR0NEMAiUtc7ZEYsq9kzValHwvgAvSM/edit?usp=sharing)
- For feedback on the basemaps, reach out to:
dchavez@utah.gov
jcowley@utah.gov
ahansgen@utah.gov
amcelwee@utah.gov

### Risks

- 

### Issue Reference

- #

**Comment by chriswnek** [comment]
Received good feedback from State Parks today on the Parksy basemap.  Added new checkbox to the Acceptance Criteria regarding labels having a 4:5:1 contrast ratio.  Also added contact info to the Notes for ongoing basemap feedback (may also start having a weekly stand-up meeting on Mondays).

Deadline has been pushed to end of May.

**Comment by chriswnek** [comment]
From our State Parks meeting on 5/12/26:
- For the Parksy basemap, the only feedback received was to try upping the contrast on the labels to help make them more readable.
- For the current "High Contrast" basemap that we're using as an accessible option, Aaron from the DXP stated that the labels should be bigger.  In order to do this, we'd need to save a copy of the basemap into our org, and then edit and publish that version.

**Comment by stdavis** [comment]
## Sprint 2

Refactor skid to update data in-place rather than truncate/append.
Submitted risk exception for prod service. It seems to have been approved but not yet acted upon.

**Comment by stdavis** [comment]
I screwed up the first risk acceptance form for prod and put the dev project. I had to re-submit a new form today: [RITM0495073](https://services.dts.utah.gov/esc?id=ticket&table=sc_req_item&sys_id=72798085cfe20b50474cfa742f851c30&view=sp)

**Comment by stdavis** [comment]
I did some work on this project this morning. It's deployed to the production cloud project and just needs a [few more steps](https://github.com/agrc/state-parks-skid/issues/12) once the production WordPress site is setup before it's ready for prime-time.
